asset_stream_manager/asset_stream_config.cc

```diff
@@ -16,83 +16,138 @@
-ABSL_DECLARE_FLAG(std::string, dev_src_dir);
-ABSL_DECLARE_FLAG(std::string, dev_user_host);
-ABSL_DECLARE_FLAG(uint16_t, dev_ssh_port);
-ABSL_DECLARE_FLAG(std::string, dev_ssh_command);
-ABSL_DECLARE_FLAG(std::string, dev_scp_command);
-ABSL_DECLARE_FLAG(std::string, dev_mount_dir);
-
-// Declare AS20 flags, so that AS30 can be used on older SDKs simply by
-// replacing the binary. Note that the RETIRED_FLAGS macro can't be used
-// because the flags contain dashes. This code mimics the macro.
-absl::flags_internal::RetiredFlag<std::string> RETIRED_FLAGS_session_ports;
-absl::flags_internal::RetiredFlag<std::string> RETIRED_FLAGS_gm_mount_point;
-absl::flags_internal::RetiredFlag<bool> RETIRED_FLAGS_allow_edge;
-
-const auto RETIRED_FLAGS_REG_session_ports =
-    (RETIRED_FLAGS_session_ports.Retire("session-ports"),
-     ::absl::flags_internal::FlagRegistrarEmpty{});
-const auto RETIRED_FLAGS_REG_gm_mount_point =
-    (RETIRED_FLAGS_gm_mount_point.Retire("gamelet-mount-point"),
-     ::absl::flags_internal::FlagRegistrarEmpty{});
-const auto RETIRED_FLAGS_REG_allow_edge =
-    (RETIRED_FLAGS_allow_edge.Retire("allow-edge"),
-     ::absl::flags_internal::FlagRegistrarEmpty{});
+#include "lyra/lyra.hpp"
 
 namespace cdc_ft {
 
-AssetStreamConfig::AssetStreamConfig() {
-  session_cfg_.verbosity = absl::GetFlag(FLAGS_verbosity);
-  session_cfg_.fuse_debug = absl::GetFlag(FLAGS_debug);
-  session_cfg_.fuse_singlethreaded = absl::GetFlag(FLAGS_singlethreaded);
-  session_cfg_.stats = absl::GetFlag(FLAGS_stats);
-  session_cfg_.quiet = absl::GetFlag(FLAGS_quiet);
-  session_cfg_.fuse_check = absl::GetFlag(FLAGS_check);
-  log_to_stdout_ = absl::GetFlag(FLAGS_log_to_stdout);
-  session_cfg_.fuse_cache_capacity = absl::GetFlag(FLAGS_cache_capacity).Size();
-  session_cfg_.fuse_cleanup_timeout_sec = absl::GetFlag(FLAGS_cleanup_timeout);
-  session_cfg_.fuse_access_idle_timeout_sec =
-      absl::GetFlag(FLAGS_access_idle_timeout);
-  session_cfg_.manifest_updater_threads =
-      absl::GetFlag(FLAGS_manifest_updater_threads);
-  session_cfg_.file_change_wait_duration_ms =
-      absl::GetFlag(FLAGS_file_change_wait_duration_ms);
-
-  dev_src_dir_ = absl::GetFlag(FLAGS_dev_src_dir);
-  dev_target_.user_host = absl::GetFlag(FLAGS_dev_user_host);
-  dev_target_.ssh_port = absl::GetFlag(FLAGS_dev_ssh_port);
-  dev_target_.ssh_command = absl::GetFlag(FLAGS_dev_ssh_command);
-  dev_target_.scp_command = absl::GetFlag(FLAGS_dev_scp_command);
-  dev_target_.mount_dir = absl::GetFlag(FLAGS_dev_mount_dir);
-}
+AssetStreamConfig::AssetStreamConfig() = default;
 
 AssetStreamConfig::~AssetStreamConfig() = default;
 
+void AssetStreamConfig::RegisterCommandLineFlags(lyra::command& cmd) {
+  session_cfg_.verbosity = 2;
+  cmd.add_argument(lyra::opt(session_cfg_.verbosity, "num")
+                       .name("--verbosity")
+                       .help("Verbosity of the log output, default: " +
+                             std::to_string(session_cfg_.verbosity)));
```

Comment:
You could reuse that constant then again here.

Reply:
Done.